ts/session/types/PubKey.ts

```diff
@@ -88,7 +88,7 @@ export class PubKey {
     const pk = value instanceof PubKey ? valAny.key : value;
 
     if (!pk || pk.length < 8) {
-      throw new Error('PubKey.shorten was given an invalid PubKey to shorten.');
+      return pk;
```

Comment:
Is this correct?

Reply:
yeah, I guess so. We probably don't want to throw at all for this, but return what we were given

Reply:
Doesn't this mean it can fail silently if we pass an empty string? Maybe we should at least log a warning or something? Or throw on `!pk` but return on `pk.length < 8`?

Reply:
changed to 
```ts
    if (!pk) {
      throw new Error('PubKey.shorten was given an invalid PubKey to shorten.');
    }

    if (pk.length < 8) {
      return pk;
    }